src/demo/demo.ejs

```diff
@@ -1,14 +1,16 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8">
+    <meta charset="utf-8" />
     <title>Onfido SDK Demo</title>
-    <meta name="viewport" content="width=device-width, minimal-ui">
-    <meta name="mobile-web-app-capable" content="yes">
-    <meta name="apple-mobile-web-app-capable" content="yes">
-    <meta name="format-detection" content="telephone=no">
+    <meta name="viewport" content="width=device-width, minimal-ui" />
+    <meta name="mobile-web-app-capable" content="yes" />
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+    <meta name="format-detection" content="telephone=no" />
     <% if (process.env.NODE_ENV !== "development"){ %>
-    <meta http-equiv="Content-Security-Policy" content="
+    <meta
+      http-equiv="Content-Security-Policy"
+      content="
       default-src 'self' https://assets.onfido.com;
       script-src 'self' https://www.woopra.com https://assets.onfido.com https://sentry.io;
```

Comment:
For local development environment, the CSP complains about missing the polyfill from `script-src`:
```suggestion
      script-src 'self' https://www.woopra.com https://assets.onfido.com https://sentry.io https://cdn.polyfill.io/v3/polyfill.min.js;
```